Source/CardStackController.m

```diff
@@ -662,4 +640,51 @@ - (void)moveCard:(CardView *)card
     [card pop_addAnimation:springAnimation forKey:@"frame"];
 }
 
+-(void)calculateMaximumTotalTitleBarHeightBeforeSearchAppears{
```

Comment:
Could you please add a space after `-` and before `{`? Just to keep formatting consistent?